packages/rocketchat-livechat/package.js

```diff
@@ -190,8 +191,10 @@ Package.onUse(function(api) {
 	api.addFiles('server/publications/livechatInquiries.js', 'server');
 	api.addFiles('server/publications/livechatOfficeHours.js', 'server');
 
-	// api
-	api.addFiles('server/api.js', 'server');
+	// REST endpoints
+	api.addFiles('server/rest/departments.js', 'server');
+	api.addFiles('server/rest/sms.js', 'server');
+	api.addFiles('server/rest/users.js', 'server');
```

Comment:
Maybe we can start using imports here. What do you think?
We can keep the `server/api.js` and just add the imports for each api file.